src/panel-app/button.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useCss } from "kremling";
+
+export default function Button({ children, ...props }) {
+  const scope = useCss(css);
+  return (
+    <button {...props} {...scope} className="button">
```

Comment:
ah, the fun of custom buttons... 

doing it this way, a person using the button can't put their own className on it, right? should maybe look at a way to combine the className we need with any className passed in.

Reply:
Umm, maybe if this were a larger feature with many more contributors. I don't see a benefit to adding that when we are currently only rendering a handful of buttons with the same style. That can be added easily enough when requirements change in the future.